static/showcases/blocked-segment.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>Pillarbox Demo - Detect blocked segment</title>
+  <link rel="icon" type="image/x-icon" href="../../img/favicon.ico">
+  <link rel="stylesheet" href="./static-showcase.scss"/>
+  <link rel="stylesheet" href="./blocked-segment.scss"/>
+</head>
+
+<body>
+<core-demo-header></core-demo-header>
+<div class="showcase-content">
+  <h2>Detect blocked segment</h2>
+  <div class="video-container">
+    <video-js id="video-element-id" class="pillarbox-js" controls crossorigin="anonymous"></video-js>
+  </div>
+
+  <button class="showcase-btn" id="close-btn">Close this window</button>
+</div>
+
+<script type="module" data-implementation>
+  // Import the pillarbox library
+  import pillarbox from '@srgssr/pillarbox-web';
+
+  // Register a plugin to display notifications on blocked segments
+  pillarbox.registerPlugin('blockedSegmentNotification', function(options) {
+    // Get a reference to the player instance
+    const player = this;
+
+    // Wait for the player to be ready
+    player.ready(() => {
+      // Create a component for displaying the blocked segment notification
+      const blockSegmentNotification = player.addChild('component', {
+        className: 'pbw-blocked-segment-notification'
+      });
+
+      // Hide the notification initially
+      blockSegmentNotification.hide();
```

Comment:
something like:
```javascript
const blockSegmentNotification = player.addChild('component', {
    className: 'pbw-blocked-segment-notification vjs-hidden'
});
```

will avoid doing:

```javascript
// Hide the notification initially
blockSegmentNotification.hide();
```

Reply:
I've added the class and a comment explaining what it does.